### PR TITLE
Nelson 0.5.7

### PR DESCRIPTION
**Pull request by Nelson-numerical-software**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22292)

### CODE REVIEW

File: manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: NelsonNumericalSoftware.Nelson
+PackageVersion: 0.5.7
+PackageLocale: en-US
+Publisher: Nelson Numerical Software
```
Comment (ItzLevvie) [inline manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml:5]:
```suggestion
Publisher: Allan CORNET
```

File: manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: NelsonNumericalSoftware.Nelson
+PackageVersion: 0.5.7
```
Comment (ItzLevvie) [inline manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml:3]:
```suggestion
PackageVersion: 0.5.7.1334
```

File: manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: NelsonNumericalSoftware.Nelson
+PackageVersion: 0.5.7
```
Comment (ItzLevvie) [inline manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.yaml:3]:
```suggestion
PackageVersion: 0.5.7.1334
```

File: manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: NelsonNumericalSoftware.Nelson
+PackageVersion: 0.5.7
```
Comment (ItzLevvie) [inline manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.installer.yaml:3]:
```suggestion
PackageVersion: 0.5.7.1334
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
needs to match Control Panel.

File: manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml (hunk 1 above)
Comment (Nelson-numerical-software) [reply manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml:5]: not as person but as organization

Comment (ItzLevvie) [reply manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml:5]: You'll have to update that in your software then.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by Nelson-numerical-software — COMMENTED** [review]
Thanks to apply your changes

File: manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: NelsonNumericalSoftware.Nelson
+PackageVersion: 0.5.7.1334
+PackageLocale: en-US
+Publisher: Nelson Numerical Software
```
Comment (ItzLevvie) [inline manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml:5]:
```suggestion
Publisher: Allan CORNET
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update your version folder as well to pass pipelines.

**Comment by Nelson-numerical-software** [comment]
> Hello @Nelson-numerical-software,
> The package manager bot determined that the metadata was not compliant.
> 
> Please verify the manifest file is compliant with the package manager [1.0 manifest specification](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md).
> Make sure the ID is of the form publisher.appname and that the folder structure is manifests\partition\publisher\appname\version.
> Note: The path and "PackageIdentifier" are case sensitive.
> Be sure to use a tool like VSCode (https://code.visualstudio.com/) to make sure the manifest YAML syntax is correct.
> 
> You could also try our [Windows Package Manager Manifest Creator Preview](https://github.com/microsoft/winget-create).
> 
> For details on the specific error, see the details link below in the build pipeline.

@ItzLevvie, I modified as you requested.

Please also notice that I do not publish as "one person" but as "an organization" !!!

**Comment by ItzLevvie** [comment]
> I modified as you requested and it fails

This is because you've not updated the version folder from 0.5.7 to 0.5.7.1334 - this must match Control Panel because Windows Package Manager uses data from Control Panel to map the application in winget list and prevent upgrade loops when upgrading the application using winget upgrade.

> Please also notice that I do not want to publish as "one person" but as "an organization" as previous versions !!!
So do I restore Publisher value as previously ?

Please recompile your software with the organization change and update the publisher and hash keys here.

Unfortunately, `Allan CORNET` is what shows up in Control Panel when installing this program and if it's changed to something else, then it may cause issues with the mapping.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see https://github.com/microsoft/winget-pkgs/pull/22292#issuecomment-887314105

**Comment by Nelson-numerical-software** [comment]
@ItzLevvie I will fix publisher in next release (https://github.com/Nelson-numerical-software/nelson/issues/480)
Currently I restored as previously and updated path.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**